ios/FluentUI.Demo/FluentUI.Demo/Demos/CommandBarDemoController.swift

```diff
@@ -156,6 +158,84 @@ class CommandBarDemoController: DemoController {
         container.layoutMargins.left = 0
         view.backgroundColor = Colors.surfaceSecondary
 
+        container.addArrangedSubview(createLabelWithText("Default"))
+
+        defaultCommandBar = CommandBar(itemGroups: createItemGroups(), leadingItemGroups: [[newItem(for: .keyboard)]])
+        defaultCommandBar?.backgroundColor = Colors.navigationBarBackground
+        container.addArrangedSubview(defaultCommandBar!)
```

Comment:
We should be avoiding force unwraps wherever possible. Better to guard/let here, perhaps?